karaf-features/pom.xml

```diff
@@ -84,9 +84,10 @@
                             </framework>
                             <features>
                                 <!-- <feature>opennms-plugins-zabbix</feature> -->
-                                <feature>zabbix-guava</feature>
-                                <feature>zabbix-jackson</feature>
-                                <feature>zabbix-netty</feature>
+                                <feature>okhttp</feature>
```

Comment:
A lot of unrelated changes in this PR, which revert some of the other work that was done.

Let's only update the async related code.